api/v6.0/report_server.thrift

```diff
@@ -12,9 +12,8 @@
 
 include "shared.thrift"
 
-namespace py codeCheckerDBAccess
-namespace js codeCheckerDBAccess
-namespace cpp cc.service.codechecker
+namespace py codeCheckerDBAccess_v60
+namespace js codeCheckerDBAccess_v60
```

Comment:
Is it a good idea to include the minor version in the namespace?

Reply:
it's not